.eslintrc.json

```diff
@@ -18,7 +18,7 @@
     "@typescript-eslint/await-thenable": "error",
     "@typescript-eslint/ban-ts-ignore": "error",
     "camelcase": "off",
-    "@typescript-eslint/camelcase": "error",
+    "@typescript-eslint/camelcase": "off",
```

Comment:
I got tired of adding all the line by line exceptions.  Fact is as we deserialize these requests into structures dictated externally, many have non camel cased properties.